README.md

```diff
@@ -289,10 +289,10 @@ Application app = client.getApplication("appId");
 SwaApplication swaApp = client.instantiate(SwaApplication.class)
     .setSettings(client.instantiate(SwaApplicationSettings.class)
     .setApp(client.instantiate(SwaApplicationSettingsApplication.class)
-      .setButtonField("btn-login")
```

Comment:
revert indent

Reply:
Doesn't the indention update look nicer? :)

Reply:
IMHO, no:
1.) it's distracting from this commit (if this wasn't a large PR, i think it would be something we could chat about)
2.) Those properties are nested under `.setApp(...`

Reply:
Ok got it